Sources/Plasma/FeatureLib/pfPython/pyGUIControlListBoxGlue.cpp

```diff
@@ -257,41 +257,41 @@ PYTHON_METHOD_DEFINITION(ptGUIControlListBox, addImageInBox, args)
 
 PYTHON_METHOD_DEFINITION(ptGUIControlListBox, addStringWithColor, args)
 {
-    char* text;
+    wchar_t* text;
     PyObject* colorObj = nullptr;
     unsigned long inheritAlpha;
-    if (!PyArg_ParseTuple(args, "sOl", &text, &colorObj, &inheritAlpha))
+    if (!PyArg_ParseTuple(args, "uOl", &text, &colorObj, &inheritAlpha))
     {
-        PyErr_SetString(PyExc_TypeError, "addStringWithColor expects a string, ptColor, and an unsigned long");
+        PyErr_SetString(PyExc_TypeError, "addStringWithColor expects a unicode string, ptColor, and an unsigned long");
```

Comment:
```suggestion
        PyErr_SetString(PyExc_TypeError, "addStringWithColor expects a string, ptColor, and an unsigned long");
```

Strings are unicode in Python 3.

Reply:
I was just modeling this message after another glue method above these functions that also takes a "u" (unicode string) like I changed these to use.... I can revert if you like -- just explaining my thought process to make it consistent with other code.